### DISCUSSION TITLE
购买一年的变为免费版，重新授权不成功。

### DISCUSSION THREAD

**Discussion by shenmz233** [getlantern/lantern-forum-cn#1023] [General]
![微信图片_20241028025749](https://github.com/user-attachments/assets/0de06178-5102-44fa-b328-060daf550ea7)

请帮忙反馈验证码 至 shenmz233@163.com 邮箱。

**Comment by MaxMKMax** [comment]
@shenmz233 您好. 好的, 已经帮您提交问题上去了. 谢谢您的耐心等待!
